extensions/azurehybridtoolkit/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "azurehybridtoolkit",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "engines": {
+    "vscode": "*",
+    "azdata": "*"
+  },
+  "main": "./out/main",
+  "activationEvents": [
+    "*"
+  ],
+  "contributes": {
+    "commands": [
```

Comment:
You should probably also contribute to the notebooks.books extension point so this shows up in the ... menu of `Provided Books`

Reply:
See https://mssqltools.visualstudio.com/SqlOpsStudioExtensions/_git/SqlOpsStudioExtensions?path=%2Fextensions%2Fsqlservernotebook%2Fpackage.json&version=GBmain&line=18&lineEnd=19&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents for an example